CHANGELOG

```diff
@@ -1,6 +1,11 @@
 django-plans changelog
 ======================
 
+0.9.1
+------
```

Comment:
We need to keep the changelog under `0.9.0` since even that `0.9.0` is not release yet.